api/pkg/apps/webapp/server.go

```diff
@@ -99,6 +100,12 @@ func NewServer(options *ServerOptions) (*Server, error) {
 		return nil, err
 	}
 
+	err = transpileTS()
+	if err != nil {
+		logrus.WithError(err).Errorf("failed to transpile typescript files")
+		return nil, err
+	}
+
```

Comment:
I think that transpiling typescript files should rather be part of the build process..